run-dlp/workflow-definitions/dlp-gcs-file.yaml

```diff
@@ -0,0 +1,65 @@
+main:
```

Comment:
Maybe rename the file from dlp-gcs-file.yaml to dlp-gcs-workflow.yaml or simply workflow.yaml? Since this is a single file, I also wonder if we need the workflow-definitions folder